io.asgardeo.tomcat.oidc.sample/src/main/resources/configuration-catalog.md

```diff
@@ -191,3 +204,67 @@ the page that the user would always be redirected to, after successful authentic
 ```
 homePage=home.jsp
 ```
+
+### Post Logout Redirect URI
+
+**Property Name:** `postLogoutRedirectURI`
+
+**Description:** This is the URI to which the user will be returned upon successfully signing out from the OpenID Provider. If not configured, the callbackURL will be used instead.
```

Comment:
returned to*

Reply:
Resolved via [bf5e232](https://github.com/asgardeo/asgardeo-tomcat-oidc-agent/pull/42/commits/bf5e2323c61ba6b2673ac881213eefe1a29e579a)